src/constants/errors.js

```diff
@@ -0,0 +1,2 @@
+export const METRICS_NOT_PROVIDED =
+  "Can not generate empty chart data. Metrics array couldn't be empty! ";
```

Comment:
Slight reword: "Cannot generate a chart with empty data. The metrics array should not be empty."